guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -46,7 +46,7 @@ There can only be one active job with a given purpose at a time.
 Click *Next* after you have entered the required information.
 . Review job details.
 You have the option to return to any part of the job wizard and edit the information.
-. Click *Run* to schedule the job for execution.
+. Click *Submit* to execute the job.
```

Comment:
I think you should keep it like this because the job doesn't need to be executed immediately based on the execution time that user selected.
```suggestion
. Click *Submit* to schedule the job for execution.
```